### ISSUE TITLE
About Your RMSE

### ISSUE THREAD

**Issue by Padfoot-Luna** [xinychen/transdim#2]
1. The RMSE in BGCP and BPTF, why do you divide it by dense_tensor[pos].shape[0] which in fact is the dim1 instead of the number of samples?
2. In CP_ALS and Tucker_ALS,  why do you calculate the RMSE on the training set ( sparse_tensor) not on the test set as you do in BGCP?

**Comment by xinychen** [comment]
This is a good question. Of course, you could replace that performance metric, and I will do that soon.
